python/scripts/json-conversion/fossa-deps-conversion.py

```diff
@@ -20,6 +20,7 @@ def saveFossaDepsJson(dictionary):
     file.close()
 
 def findRpmTarVendoredDependencies():
+    reg = re.compile('[a-zA-Z0-9\.-_]')
```

Comment:
Need to import re at the beginning of the script.

Reply:
my bad!